ups/eupspkg.cfg.sh

```diff
@@ -0,0 +1,6 @@
+build(){
+    scons opt=3 prefix=$PREFIX version=$VERSION
+}
+install(){
+    scons opt=3 install prefix=$PREFIX version=$VERSION
+}
```

Comment:
Removed. I figured out it would not be necessary after the fact of creating it, but just had not gone back and removed it. I was playing around with different opt levels and debug for a while.